training/training_code/train_ner.py

```diff
@@ -23,9 +23,11 @@
 from tensorflow.python.keras.layers import concatenate
 from tensorflow.python.keras.models import Model
 from tensorflow.python.keras.optimizers import Adam
-
+from tensorflow.python.keras.callbacks import ModelCheckpoint
```

Comment:
Have we checked that the new model training code still works on WML - since we still need to target it.

Specifically, though this should be TF 2.x compatible code, does it still work on 1.15 for WML?

Reply:
I tested locally with TF 2.1 and TF 1.15.

I did not check with WML. Will run the WML test as well